website/docs/guides/provision_appliances.markdown

```diff
@@ -0,0 +1,184 @@
+---
+layout: "appgatesdp"
+page_title: "Provisioning appliances"
+sidebar_current: "docs-appgatesdp-guide-state_migrate"
+description: |-
+  Provisioning appliances
+---
+
+## Provisioning appliances
+
+
+
+### cz-seed
+
+Appliances are provisioned with the [seed file](https://sdphelp.appgate.com/adminguide/v5.3/new-appliance.html?anchor=manual-seeding), we use `cz-seed` to the provision the appliances. `cz-seed` is a built-in program included on every appliance.
+This is a non interactive version of [cz-setup](https://sdphelp.appgate.com/adminguide/v5.3/controller-cz-setup.html). 
```

Comment:
You could expand here to explain that this is only for two cases:
- Seeding a first controller
- Seeding initial configuration on an additional appliance, before receiving the actual seed from an existing controller.
 This is required is some cases to get network configuration or SSH authentication setup in order to perform the actual seeding and join an existing controller group.